contracts/strategies/BaseStrategy.sol

```diff
@@ -54,13 +54,15 @@ abstract contract BaseStrategy is IStrategy, Transfer {
         _;
     }
 
+    /// @notice Modifier to check if the pool is inactive
     modifier onlyInactivePool() {
         if (poolActive) {
             revert BaseStrategy_POOL_ACTIVE();
         }
         _;
     }
 
+    /// @notice Modifier to check if the pool is initialized
```

Comment:
Can we remove this, it's added in the natspec branch

Reply:
```suggestion
```